### PR TITLE
Tinkering

### PR DESCRIPTION
**Pull request by SignpostMarv**
tinkered a little.

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.4/phpunit.xsd"
+         colors="true"
+         verbose="true"
+         bootstrap="vendor/autoload.php">
+    <testsuites>
+        <testsuite name="reCAPTCHA Test Suite">
```
Comment (jfschmakeit) [inline phpunit.xml.dist]: This looks like the wrong name. What about "php-photoslibrary Test Suite"?

File: phpunit.xml.dist [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        </testsuite>
+    </testsuites>
+    <filter>
+        <whitelist>
+            <directory suffix=".php">src/Google/Photos/Library/V1/</directory>
+        </whitelist>
+    </filter>
+    <logging>
+        <log
+            type="coverage-text"
```
Comment (jfschmakeit) [inline phpunit.xml.dist]: Formatting nit: shouldn't the first parameter be on the previous line?

File: tests/Unit/V1/FiltersBuilderTest.php [hunk 3]
```diff
@@ -78,7 +78,7 @@ public function testBuildWithAllFields()
 
     public function testAddInvalidIncludedCategory()
     {
-        $this->expectException(\InvalidArgumentException::class);
+        $this->setExpectedException(\InvalidArgumentException::class);
```
Comment (jfschmakeit) [inline tests/Unit/V1/FiltersBuilderTest.php:81]: `setExpectedException` has been deprecated, it looks like this should just be `expectException` as per: https://phpunit.readthedocs.io/en/7.3/writing-tests-for-phpunit.html#testing-exceptions 

Can you clarify why this should be `setExpectedException`?

**Review by jfschmakeit — CHANGES_REQUESTED** [review]
Thanks for submitting this pull request, please see some comments inline.

File: phpunit.xml.dist (hunk 2 above)
Comment (SignpostMarv) [reply phpunit.xml.dist]: This was copy/paste from an existing file, I can reformat it later :)

File: phpunit.xml.dist (hunk 1 above)
Comment (SignpostMarv) [reply phpunit.xml.dist]: copypasta, my bad. will change later :)

File: tests/Unit/V1/FiltersBuilderTest.php (hunk 3 above)
Comment (SignpostMarv) [reply tests/Unit/V1/FiltersBuilderTest.php:81]: composer.json requires phpunit/phpunit 4.8 as a minimum (likely to satisfy the php 5.5 support requirement).

[setExpectedException()](https://github.com/sebastianbergmann/phpunit/blob/18e5f52e8412d23e739f7d8744e177039860e800/src/Framework/TestCase.php#L462) is the appropriate method to use across phpunit `^4.8|^5.0`, as [expectException() wasn't introduced until 5.2](https://github.com/sebastianbergmann/phpunit/blob/09c74ffc299a6bf5e6d1444cdc44b4f98f5826da/src/Framework/TestCase.php#L507)

If the minimum php version is raised to 5.6, we could change the phpunit constraint to `^5.2` & restore the `expectException` method.

File: phpunit.xml.dist (hunk 1 above)
Comment (SignpostMarv) [reply phpunit.xml.dist]: resolved in 5b5e9c6

File: phpunit.xml.dist (hunk 2 above)
Comment (SignpostMarv) [reply phpunit.xml.dist]: resolved in 5b5e9c6

**Comment by SignpostMarv** [comment]
I'm wondering why this was closed ?

**Comment by jfschmakeit** [comment]
It looks like GitHub automatically closed this pull request when we renamed the default branch. Sorry I missed your reply - this looks perfectly reasonable. Would you be able to rebase it back onto the `main` branch and @ me in the pull request so we can get this reviewed and submitted? 

Cheers!

**Pull request closed without merging**